ios/StripeSdk.swift

```diff
@@ -452,6 +452,44 @@ class StripeSdk: RCTEventEmitter, STPApplePayContextDelegate, STPBankSelectionVi
         }
     }
     
+    @objc(createToken:resolver:rejecter:)
+    func createToken(
+        params: NSDictionary,
+        resolver resolve: @escaping RCTPromiseResolveBlock,
+        rejecter reject: @escaping RCTPromiseRejectBlock
+    ) -> Void {
+        let address = params["address"] as? NSDictionary
+        
+        if let type = params["type"] as? String {
+            if (type != "Card") {
+                reject(CreateTokenErrorType.Failed.rawValue, type + " type is not supported yet", nil)
+            }
+        }
+        
+        let cardFieldUIManager = bridge.module(forName: "CardFieldManager") as? CardFieldManager
+        let cardFieldView = cardFieldUIManager?.getCardFieldReference(id: CARD_FIELD_INSTANCE_ID) as? CardFieldView
+        
+        guard let cardParams = cardFieldView?.cardParams else {
+            reject(CreateTokenErrorType.Failed.rawValue, "You must provide card details", nil)
+            return
+        }
+        
+        let cardSourceParams = STPCardParams()
+        cardSourceParams.number = cardParams.number
+        cardSourceParams.cvc = cardParams.cvc
+        cardSourceParams.expMonth = UInt(truncating: cardParams.expMonth ?? 0)
+        cardSourceParams.expYear = UInt(truncating: cardParams.expYear ?? 0)
```

Comment:
`null` instead of `0` please

Reply:
@thorsten-stripe  actually it's only for type checking purposes, this code will never run when card details are not completed, also this field is not nullable so I can't set `null` fallback